docs/pages/api-reference/liveblocks-react-ui.mdx

```diff
@@ -12,6 +12,201 @@ to build collaborative experiences. Read our
+}
+```
+
+##### Show placeholder content in new chats
+
+In chats without messages, you can display placeholder content to guide the
+user...
+
+{/* TODO the API we figure out */}
+
+{/* TODO show initial suggestions if it's supported */}
+
+##### Adjusting the chat’s width
+
+{/* TODO add in name of layout below */}
+
+When using the default `TODO` layout, it’s possible to adjust the chat’s width
+by setting the `--lb-ai-chat-container-width` CSS variable. This allows the
+chat’s scroll window to stay full width, whilst keeping the composer and
+messages centered in the middle.
+
+```css
+.lb-root {
+  --lb-ai-chat-container-width: 600px;
+}
+```
+
+You can also set this property directly on the `AiChat` component.
+
+```tsx
+import { CSSProperties } from "react";
+import { AiChat } from "@liveblocks/react-ui";
+
+function Chat() {
+  return (
+    <AiChat
+      chatId="my-chat-id"
+      // +++
+      style={{ "--lb-ai-chat-container-width": 600 } as CSSProperties}
+      // +++
+    />
+  );
+}
+```
+
+##### Compact layout mode
+
+An alternate full-width layout mode is available, ideal for smaller UI
+components such as pop-up windows. Compact layout mode removes the shadow and
+padding on the composer, and displays a border above it.
+
+{/* TODO mention how to use the other layout mode */}
+
+##### List the user’s chats and switch between them
+
+You can display a list of all chats created by the current user with
+[`useAiChats`](/docs/api-reference/liveblocks-react#useAiChats). For example,
+you can render a list of buttons that allow you to switch between chats. In each
+button, you can display the chat’s automatically generated title, as seen below.
+Chats can be deleted with
+[`useDeleteAiChat`](/docs/api-reference/liveblocks-react#useDeleteAiChat).
+
+```tsx
+import { useState } from "react";
+import { AiChat } from "@liveblocks/react-ui";
+import { useAiChats } from "@liveblocks/react";
+
+function Chats() {
+  // +++
+  const { chats, error, isLoading } = useAiChats();
+  const [chatId, setChatId] = useState();
+  const deleteChat = useDeleteAiChat();
+  // +++
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  }
+
+  return (
+    <div style={{ display: "flex" }}>
+      <ul>
+        // +++
+        {chats.map((chat) => (
+          <li key={chat.id}>
+            <button onClick={() => setChatId(chat.id)}>
+              {chat.title || "Untitled"}
+            </button>
+            <button onClick={() => deleteChat(chat.id)}>❌</button>
+          </li>
+        ))}
+        // +++
+      </ul>
+      // +++
+      <AiChat chatId={chatId} />
+      // +++
+    </div>
+  );
+}
+```
+
+##### Display the chat’s title
+
+{/* TODO mention metadata too, if supported */}
+
+Each chat has a `title` property, generated by AI after it reads the user’s
+first message. You can render this alongside your chat with
+[`useAiChat`](#TODO).
```

Comment:
Should we mention the component will automatically re-render when the title is auto-updated or do you think it's implied?